packages/core/src/components/Card/Card.tsx

```diff
@@ -61,8 +61,7 @@ export const HvCard = (props: HvCardProps) => {
         "HvIsCardGridElement",
         classes.root,
         css({
-          backgroundColor:
-            (bgcolor && theme.colors[bgcolor]) || theme.card.backgroundColor,
+          backgroundColor: getColor(bgcolor),
```

Comment:
I think we should include these improvements & missing `useDefaultProps` in a separate PR, to reduce the scope & changes of this huge PR